.github/scripts/compile.sh

```diff
@@ -134,7 +134,7 @@ do_build () {
 # Do a regular build without asan or tsan.
 do_build "" OFF
 # and install Proton Python
-python3 -m pip install --prefix="$PROTON_INSTALL_DIR/usr" "$(find "$PROTON_BUILD_DIR/python/dist/" -name 'python-qpid-proton*.tar.gz')"
+python3 -m pip install --prefix="$PROTON_INSTALL_DIR/usr" "$(find "$PROTON_BUILD_DIR/python/{pkgs,dist}/" -name 'python-qpid-proton*.tar.gz')"
```

Comment:
The {pkgs,dist} syntax is not recognised by find or expanded by the shell and causes the container build to fail.
I think you can just use multiple directories directly:
"$PROTON_BUILD_DIR/python/dist/" "$PROTON_BUILD_DIR/python/pkgs/"

Reply:
Ouch. I guess I should've looked after pushing, if it works. I got distracted by something else.

That `{pkgs,dist}` would've been expanded by shell if I removed the quotes around it. (I tried it in shell without quotes, and thought that they don't make a difference.) Ok, I'll put there the dirs directly, that is resilient towards spaces in the paths...

Reply:
> "$PROTON_BUILD_DIR/python/dist/" "$PROTON_BUILD_DIR/python/pkgs/"

This would not work either, because find fails if given nonexisting path. I'm going to give it `"$PROTON_BUILD_DIR/python`, that seems to work ok.